content/docs/integrations/postgres.md

```diff
@@ -1,19 +1,13 @@
 ---
 title: Run a Postgres App
-redirectFrom:
-  - /docs/quickstart/postgres
 ---
 
-Since Neon compute is regular PostgreSQL, any PostgreSQL app will run out of the box.
-You can also use all the standard client utilities like `psql` and `pg_dump`, client libraries and drivers to connect.
+A Neon compute runs regular PostgreSQL, which means that you run any PostgreSQL app with Neon.
 
-All connections to PostgreSQL are via an endpoint hosted at `pg.neon.tech`.
+You can also connect to Neon using standard PostgreSQL client utilities such as `psql` and `pg_dump`, standard client libraries, and drivers.
 
-Try connecting to `pg.neon.tech` using the following command in your terminal:
+All connections to Neon PostgreSQL are via an endpoint hosted at `pg.neon.tech`.
```

Comment:
I'm not sure that this is the only endpoint we support.
Not sure if we have the same proxy for the passwordless auth and for the normal connections.
Please double check?

Reply:
I have set this topic to isDraft to exclude it from Algolia search, so users will not see this content. I will eventually remove the file entirely. 
I have an issue open document the endpoints that we support: #180